.nais/deploy-catbox.yml

```diff
@@ -52,3 +54,5 @@ spec:
   env:
     - name: "PENSJONSKALKULATOR_BACKEND"
       value: "http://pensjonskalkulator-backend"
+    - name: "IDPORTEN_OBO_ISSUER"
```

Comment:
Jeg ville kanskje ha kalt denne `TOKEN_X_OBO_AUDIENCE` (det er snakk om hvilken audience som TokenX skal utstede token for, så "issuer" blir litt misvisende her).